apps/condo/domains/meter/tasks/importMeters.js

```diff
@@ -146,8 +148,18 @@ async function importMeters (taskId) {
             await failWithErrorFile(context, taskId, errorFileContent, format)
         }
     } catch (err) {
-        const errorMessage = get(err, 'errors[0].extensions.messageForUser', get(err, 'message'))
-            || 'not recognized error'
+        let errorMessage = get(err, 'errors[0].extensions.messageForUser', get(err, 'message'))
+
+        if (errorMessage === ROWS_COUNT_LIMIT_EXCEEDED) {
```

Comment:
Could you please add a test for this error case?

Reply:
Added